egs/librispeech/ASR/pruned_transducer_stateless4/train.py

```diff
@@ -868,7 +868,7 @@ def run(rank, world_size, args):
     model.device = device
 
     if rank == 0:
-        model_avg.to(device)
+        model_avg.to(torch.device("cpu"))
         model_avg.device = device
 
```

Comment:
These lines can be removed if you want to keep `model_avg` on CPU.

Reply:
OK.